source/geometry_model/interface.cc

```diff
@@ -326,6 +326,14 @@ namespace aspect
       std::get<dim>(registered_plugins).write_plugin_graph ("Geometry model interface",
                                                             out);
     }
+
+    template <int dim>
+    void
+    Interface<dim>::make_periodicity_constraints(AffineConstraints<double> &constraints) const
```

Comment:
```suggestion
    Interface<dim>::make_periodicity_constraints(AffineConstraints<double> &/*constraints*/) const
```

this fixes the error message about the unused argument.